src/content/platform-integration/web/building.md

```diff
@@ -4,174 +4,125 @@ description: Instructions for creating a Flutter app for the web.
-  You can install [Visual Studio Code][],
-  [Android Studio][], [IntelliJ IDEA][].
-  Also [install the Flutter and Dart plugins][]
-  to enable language support and tools for refactoring,
-  running, debugging, and reloading your web app
-  within an editor. See [setting up an editor][]
-  for more details.
-
-[Android Studio]: {{site.android-dev}}/studio
-[IntelliJ IDEA]: https://www.jetbrains.com/idea/
-[Visual Studio Code]: https://code.visualstudio.com/
-
-
-For more information, see the [web FAQ][].
+Before you can build a web application with Flutter,
+make sure that you have the [Flutter SDK][] and a web browser installed.
+See the [Set up web development for Flutter][Setup-web] instructions
+for details.
 
-## Create a new project with web support
+## Set up a Flutter project
 
-You can use the following steps
-to create a new project with web support.
+To set up your project, you can create a new Flutter project or add web support
+to an existing project.
 
-### Set up
+### Create a new project
 
-Run the following commands to use the latest version of the Flutter SDK:
+To create a new app that includes web support, run the following command:
 
 ```console
-$ flutter channel stable
-$ flutter upgrade
+$ flutter create my_app 
 ```
 
-:::warning
-Running `flutter channel stable` replaces your current version of Flutter
-with the stable version and can take time if your connection is slow.
-After this, running `flutter upgrade` upgrades your install to the latest
-`stable`.  Returning to another channel (beta or master) requires calling
-`flutter channel <channel>` explicitly.
-:::
+### Add web support to an existing project
 
-If Chrome is installed,
-the `flutter devices` command outputs a `Chrome` device
-that opens the Chrome browser with your app running,
-and a `Web Server` that provides the URL serving the app.
+If you already have a project, run the `flutter create` command in your project directory:
 
 ```console
-$ flutter devices
-1 connected device:
-
-Chrome (web) • chrome • web-javascript • Google Chrome 88.0.4324.150
+$ flutter create . --platforms web
 ```
 
-In your IDE, you should see **Chrome (web)** in the device pulldown.
+This will create a `web/` directory containing the web assets used to bootstrap
+and run your Flutter app. 
 
-### Create and run
+## Run your app
 
-Creating a new project with web support is no different
-than [creating a new Flutter project][] for other platforms.
+Select [Chrome][] as your app's target device to run and debug a Flutter web app:
 
-#### IDE
+```console
+$ flutter run -d chrome
+```
 
-Create a new app in your IDE and it automatically
-creates iOS, Android, [desktop][], and web versions of your app.
-From the device pulldown, select **Chrome (web)**
-and run your app to see it launch in Chrome.
+You can also choose Chrome as a target device in your IDE.
 
-#### Command line
+You can also use the `edge` device type on Windows, or use `web-server` to
```

Comment:
```suggestion
If you prefer, you can use the `edge` device type on Windows, or use `web-server` to
```
Trying to avoid "You can also" in two consecutive lines of text.